tokens/tokens.md

```diff
@@ -0,0 +1,191 @@
+
+![Tokens Menu](section.png)
+
+The default interface of the Tokens API page shows all the currently constructed tokens, both expired and not.
+
+![Empty Tokens](empty.png)
+
+When creating a new token you will need to provide a name, description, set of permissions, and an optional expiration date.
+
+![Token Expiration](expires.png)
+
+Permissions can be selected in groups or as fine controls.  For example, you can select the "Search and Data" group which will automatically fill in all the API permissions required to interact with the search system.  This includes executing searches, getting the search history, downloading, saving, and attaching to searches.  It is also possible to pick specific functionality within search groups such as the ability to read actionables but not write them.  This example shows a token that is subscribed to the "Search and Data" group and also allows for reading Actionables and Extractors but cannot write either.
+
+![Populated Token](populated.png)
+
+Once appropriate permissions are selected go to the bottom of the permissions page and select an optional expiration; you can pick a relative date or a specific timestamp.  After that timestamp the token will still be available in the API Tokens interface but it will not be usable.  This means you can optionally refresh the token and extend its expiration or delete it at your leisure.
+
+After populating your new token with a name, description, permissions, and an optional expiration it is time to actually make the token by clicking the `Generate` button.  At this point Gravwell will create the token and provide it to you.
+
+![Generated Token](popup.png)
+
+NOTE: Make sure to record the token, Gravwell will never again provide that token to you, this is your one and only chance to record it.
+
+## Token Permissions
+
+Token permissions are defined using specific allowances, you select exactly which functionality a given token is allowed to perform.  The Gravwell user interface provides some nice features to let you select groups of permissions that might be logically related, but in the end each token must declare exactly what APIs and systems it is allowed to access.  Most permissions are divided into read/write components.  This means that you can create a token that can read resources but not write them, or a token that can read the state of automation scripts but not create, update, or schedule them.
+
+Permissions on tokens are an overlay on the users existing permissions.  This means that if the current user cannot access an API or feature, then the token cannot either.  Tokens can only restrict access, they cannot grant access that a user does not currently have.
+
+NOTE: The `Token write` permission can be particularly dangerous, if you grant a token the ability to write to the token API it can create new tokens with any permission it wants.  Token permissions are not transitive, tokens that create other tokens do not inherit their parents permission set.
```

Comment:
```suggestion
NOTE: The `Token write` permission can be particularly dangerous. If you grant a token the ability to write to the token API it can create new tokens with any permission it wants.  Token permissions are not transitive, tokens that create other tokens do not inherit their parent's permission set.
```